x-pack/plugins/security_solution/public/management/components/effected_policy_select/effected_policy_select.tsx

```diff
@@ -71,6 +72,7 @@ export type EffectedPolicySelectProps = Omit<
   isGlobal: boolean;
   isPlatinumPlus: boolean;
   description?: string;
+  arePoliciesLoading?: boolean;
```

Comment:
This might be unnecessary. The Props already have a `isLoading` prop that it inherits from `EuiSelectable`. I would suggest using that instead. Or if you keep this prop instead, then remove the `isLoading` prop from the list of props in this component.